git-annex-remote-rclone

```diff
@@ -89,6 +89,18 @@ ask () {
 	fi
 }
 
+GREP () {
+	set +e
+	out=$(grep "$@")
+	rc=$?
+	set -e
+	# Replace explicit newline since we must provide 1 line DEBUG
+	# Fancy sed is from Example 5 of https://linuxhint.com/newline_replace_sed
+	# which worked on Linux and OSX.
+	out_safe=$(echo "$out" | sed -ne 'H;${x;s/\n/\\n/g;s/^,//;p;}')
```

Comment:
If someone sees a better way to sanitize here instead of such complicated `sed` line -- I would be interested to hear!!!

Reply:
I think `tr` is cross-platform.

Reply:
With tr you can't replace one character with two AFAIK

Reply:
True :(